basic/mobile_io/2a_feedback.py

```diff
@@ -0,0 +1,48 @@
+import hebi
+from time import sleep, time
+from matplotlib import pyplot as plt
+
+lookup = hebi.Lookup()
+
+# Wait 2 seconds for the module list to populate
+sleep(2.0)
+
+family_name = "HEBI" 
+module_name = "Cal's iPhone"
```

Comment:
Make sure to change this back to what other examples are using (for consistency)